src/test.js

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import firebase from "firebase/app";
+import "firebase/firestore";
+import fb from "./lib/firebase.js";
+
+class Items extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      itemName: "",
+      dbItems: []
+    };
+  }
+
+  captureInput = e => {
+    this.setState({
+      [e.target.name]: e.target.value
+    });
+    console.log(this.state);
+  };
+
+  addItem = e => {
+    e.preventDefault();
+    let db = firebase.firestore();
+    // db.settings({
+    //     timestampsInSnapshots: true
+    // });
+    const itemRef = db.collection("items").add({
```

Comment:
I'm not too familiar with firebase. Will "itemRef" be used elsewhere once the app grows?